src/humancellatlas/data/metadata/helpers/dss.py

```diff
@@ -9,12 +10,16 @@
 from humancellatlas.data.metadata import JSON
 
 logger = logging.getLogger(__name__)
+logging.captureWarnings(True)
```

Comment:
I have a golden rule: Library code should not touch the logging configuration. That should always be a responsibility of the application using the library. 

The more general form of this rule is that importing a module should have minimal or no observable side-effects.

I'm leaning towards removing these two lines. In fact, I think where they should go is the README in a section called Logging. Along with a sentence or two that describes the effect of these two lines and why they could be useful.

Reply:
That makes sense to me. 

In fact, revisiting the 2 lines now, now I prefer to replace the `warnings.warn()` just with simple `logging.warning()` which has the same effect and reduces the overhead of using `warnings.simplefilter('always', PendingDeprecationWarning)` to control different warning types. Besides, since `logging.WARNING > logging.INFO`, for most of the cases, without tweaking, it will appear to users' stdout naturally. What do you think? @hannes-ucsc